src/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
```

Comment:
Lets also add the jquery code. not sure if it will work like we expect it though